doc/javascript/application.js

```diff
@@ -11589,27 +11589,27 @@ if ( typeof define === "function" && define.amd && define.amd.jQuery ) {
       e.preventDefault();
       try {
         parent.frames.list.$('#search input').focus().select();
-      } catch (_error) {}
+      } catch (undefined) {}
       try {
         return $('#search input').focus().select();
-      } catch (_error) {}
+      } catch (undefined) {}
     });
     key('esc', function() {
       try {
         parent.frames.list.$('#search input').blur();
         parent.frames.main.$('#help').hide();
         parent.frames.main.$('#fuzzySearch').hide();
-      } catch (_error) {}
+      } catch (undefined) {}
```

Comment:
__[jshint]__
*  Expected an identifier and instead saw 'undefined'.

*  Expected ')' and instead saw 'undefined'.

*  Too many errors. (98% scanned).